app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
 <resources>
+    <string name="app_name">WSM</string>
     <string name="locate">vi</string>
+
+    <!-- Common -->
+    <string name="error">Lỗi</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.error` appears to be unused
 Unused resources make applications larger and slow down builds.